modules/ROOT/pages/fd-share-to-studio.adoc

```diff
@@ -73,7 +73,10 @@ The connection being used by the configuration is the connection that is selecte
 .. Specify the required connection information.
 .. Repeat the three previous steps for each configuration that is marked by a red circle.
 
-
+[NOTE]
+====
+Do not import into Flow Designer a JAR file that was exported from Anypoint Studio. Flow Designer does not support all of the features of Mule applications that are supported in Anypoint Studio.
```

Comment:
that are supported in Anypoint Studio. > that Anypoint Studio supports. (not a big deal here, but maybe it's worth avoiding the passive voice?)

Reply:
@Cristian-Venticinque : The passive voice is not inherently evil. Sometimes, it's less of a burden for readers to imagine something being done, than to imagine something doing something else. In such cases, the meaning is clear, the sentence flows better, and the reader (apprised of the necessary info) can move on without effort.